api/applications/applications.proto

```diff
@@ -53,7 +53,8 @@ message ListApplicationsResponse {
 }
 
 message GetApplicationRequest {
-    string application_name = 1; // The name of an application
```

Comment:
I suspect these would be better as just `name` and `namespace` as you're asking for the `Application`

Reply:
Agreed! Changed